src/feature/dashboard/presentation/dashboard.screen.tsx

```diff
@@ -1,10 +1,280 @@
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  Paper,
+  Grid,
+  TextField,
+  Button,
+  TableContainer,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+  Modal,
+  Box,
+  IconButton,
+  Snackbar,
+  Tooltip,
+} from '@mui/material';
 import { Paperbase } from 'core/paperbase/presentation';
-import { type JSX } from 'react';
```

Comment:
Why you remove the JSX type from here? Now returned type is taken from the global scope, which is deprecated and need to be taken from react